### ISSUE TITLE
Suggestions not showing up at App Drawer 

### ISSUE THREAD

**Issue by Yograt** [Deletescape-Media/Lawnchair#4183]
### Describe the bug

Suggestions missing when feature is present at system and pixel launcher!

### Steps to reproduce

1. Go to Home setting
2. go to app drawer
3. See the suggestions feature missing

### Expected behavior

No suggestions option.

### Screenshots

![Screenshot_20240403-192819_Pixel Launcher](https://github.com/LawnchairLauncher/lawnchair/assets/107334627/df1195e1-1662-4848-85d7-ad156285fe57)
![Screenshot_20240403-192608_Pixel Launcher](https://github.com/LawnchairLauncher/lawnchair/assets/107334627/a0592167-0768-44a8-ae41-726cbfb422f5)

### Device information

- Device: Redmi Note 11 NFC (spesn)
- OS: Android 14 February Patch
- App version: Lawnchair 14 Beta 1

### Additional context

None

**Comment by AndrewDallas** [comment]
I feel this would be a feature and not a bug. I don't believe Lawnchair ever had this feature like the pixel launcher.

**Comment by Gungrave223** [comment]
> I feel this would be a feature and not a bug. I don't believe Lawnchair ever had this feature like the pixel launcher.

I can confirm that Lawnchair 100% had this feature... I had it working on the dev build this morning prior to replacing with the beta build.

**Comment by kymerea** [comment]
> > I feel this would be a feature and not a bug. I don't believe Lawnchair ever had this feature like the pixel launcher.
> 
> I can confirm that Lawnchair 100% had this feature... I had it working on the dev build this morning prior to replacing with the beta build.

Really? I never saw this feature in any of the Dev builds not the current betas. It's something I actually suggested a little while back as well. Are you sure you have the suggested apps in the _app drawer_?

**Comment by Yograt** [comment]
> > > I feel this would be a feature and not a bug. I don't believe Lawnchair ever had this feature like the pixel launcher.
> > 
> > I can confirm that Lawnchair 100% had this feature... I had it working on the dev build this morning prior to replacing with the beta build.
> 
> Really? I never saw this feature in any of the Dev builds not the current betas. It's something I actually suggested a little while back as well. Are you sure you have the suggested apps in the _app drawer_?

Yes!! There's supposed to be one at app drawer and an admin at telegram has confirmed it for me and told me to open this issue.

**Comment by Gungrave223** [comment]
So mine just literally started back working... I haven't done anything extra on beta 2...

Idk why it showed back up in my app drawer... but I'm happy.

[
![Screenshot_20240404-120223](https://github.com/LawnchairLauncher/lawnchair/assets/8568496/0c979931-6204-4223-99fc-75f20ce43b64)
](url)

**Comment by Bennett-69** [comment]
> I can confirm that Lawnchair 100% had this feature... I had it working on the dev build this morning prior to replacing with the beta build.

Yep, same here. Unlike yours, mine has yet to spontaneously reappear yet. I wonder if it takes a while to gather enough usage data to *make* suggestions of frequently-used apps? 🤔

**Comment by Bennett-69** [comment]
> So mine just literally started back working... I haven't done anything extra on beta 2...
> 
> Idk why it showed back up in my app drawer... but I'm happy.

Hmm, did you happen to install QuickSwitch recently? I discovered that doing so turns Lawnchair into a System app, which happened to fix my Google Feed and (possibly coincidentally) my app drawer suggestions. That would make theoretical sense, at least - as a System app, Lawnchair may gain default access to app usage details that it otherwise was denied? 🤔

**Comment by Gungrave223** [comment]
> > So mine just literally started back working... I haven't done anything extra on beta 2...
> > Idk why it showed back up in my app drawer... but I'm happy.
> 
> Hmm, did you happen to install QuickSwitch recently? I discovered that doing so turns Lawnchair into a System app, which happened to fix my Google Feed and (possibly coincidentally) my app drawer suggestions. That would make theoretical sense, at least - as a System app, Lawnchair may gain default access to app usage details that it otherwise was denied? 🤔

Nope...I still have to use the aid bridge for the feed and I don't have quickswitch installed since Lawnchair isn't compatible with it on the latest pixel builds.

**Comment by kymerea** [comment]
@Gungrave223 can we have a screenshot of you 'App Drawer' section in the lawnchair settings? I'm seeing nothing about suggested apps nor have I ever gotten the suggested app section in both the dev builds and the beta builds

**Comment by Gungrave223** [comment]
> @Gungrave223 can we have a screenshot of you 'App Drawer' section in the lawnchair settings? I'm seeing nothing about suggested apps nor have I ever gotten the suggested app section in both the dev builds and the beta builds


https://github.com/LawnchairLauncher/lawnchair/assets/8568496/45121389-0d86-4af9-a007-386c51282bb0

**Comment by kymerea** [comment]
Here's what I see, I don't see any suggestions 

![Screenshot_20240406-120315876](https://github.com/LawnchairLauncher/lawnchair/assets/62679764/a362dc22-2c51-4fbf-890a-20152128b8bc)

**Comment by lluni** [comment]
Can confirm on Pixel 8 Pro (Android 14, april build) using Lawnchair 14 Beta 2, I also don't see a suggestions option. This phone is not rooted, if that matters.

**Comment by TheGamer1002** [comment]
Bump, 1.1.14 (14.Dev.#1088), Pixel 6 Pro, not rooted, Android 15. It was working fine in 12.1.0 Alpha 4.

**Comment by shanemd** [comment]
I'm also unable to see suggestions in the app drawer. No option for it the same as ProjWyrd's screenshot above.

Lawnchair 14.Beta 3
Pixel 5
Stock Rom Android 14 (build UP1A.231105.001)
Rooted with Magisk 28.1

**Comment by Son-Vegitto** [comment]
Hello, 

I am using the latest nightly (15 dev) and I do not see the option for the App Suggestions row in the App Drawer. Was this feature removed?

**Comment by Gungrave223** [comment]
> Hello,
> 
> I am using the latest nightly (15 dev) and I do not see the option for the App Suggestions row in the App Drawer. Was this feature removed?

So it needs to have proper permission to be set in order to get it to show up... so you would need quick switch which only works on up to Android 15.

So if you're rooted on android 15... you should get the option if you set Lawnchair as the recent provider in quick switch.

**Comment by Son-Vegitto** [comment]
Oh, so root is required?

**Comment by Son-Vegitto** [comment]
> > Hello,
> > I am using the latest nightly (15 dev) and I do not see the option for the App Suggestions row in the App Drawer. Was this feature removed?
> 
> So it needs to have proper permission to be set in order to get it to show up... so you would need quick switch which only works on up to Android 15.
> 
> So if you're rooted on android 15... you should get the option if you set Lawnchair as the recent provider in quick switch.

Oh, so root is required?

**Comment by SuperDragonXD** [comment]
Hello! I'm going to close this issue in favor of #4235, as it is more preferable to have a fallback option that doesn't require OEMs that support Android System Intelligence's suggestions feature (which can be unpredictable, as seen here) or root.
